src/commands/mod.rs

```diff
@@ -59,6 +62,12 @@ pub enum Command {
     /// Signs a ManageNeuron message to submit a UpgradeSnsControlledCanister
     /// proposal.
     MakeUpgradeCanisterProposal(make_upgrade_canister_proposal::MakeUpgradeCanisterProposalOpts),
+    /// Prints a summary with general info about the sns.
+    SnsSummary(sns_summary::SnsSummaryOps),
```

Comment:
SnsSummary and SnsCanistersSummary don't need the 'Sns' prefix.

Reply:
You are right, sns sounds redundant